### DISCUSSION TITLE
Have a headless server

### DISCUSSION THREAD

**Discussion by Tsht** [Thorium-Sim/thorium-nova#14] [Ideas] [answered]
Idea is to have a possibility to run the server on dedicated headless (command line only) environment.
This allows to have a running server, constantly running if needed, without needing any fancy graphical features and should allows easy play over the internet on servers which does not require that much resources.

**Comment by alexanderson1993** [comment, marked as answer]
Issue created here: https://github.com/Thorium-Sim/thorium-nova/issues/17

It's not a trivial thing if we wanted to be thorough, since we would have to create binaries for each platform and do macOS code signing and notarization. But that's already done for the Electron binary, so most of the ground work is already there.
